assets/app/game_manager.rb

```diff
@@ -115,7 +115,8 @@ def enter_game(game)
       end
 
       store(:game_data, game_data.merge(loaded: true), skip: true)
-      store(:app_route, hs_url(game, game_data)) unless @app_route.include?(hs_url(game, game_data))
+      game_url = hs_url(game, game_data)
+      `window.location = #{game_url}` unless @app_route.gsub('%20', ' ').include?(game_url)
```

Comment:
For hotseat games; this forces the page to reload, and includes the game title in the URL so that when the client sends the server the request, the server knows if a game js needs to be inclued in the prerendered HTML; see https://github.com/tobymao/18xx/pull/3679/files#r570001184